elasticapm/contrib/serverless/aws.py

```diff
@@ -171,9 +167,18 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                         break
             except AttributeError:
                 pass
+            elasticapm.set_context(
+                lambda: get_data_from_response(
+                    self.response, status_code, capture_headers=self.client.config.capture_headers
+                ),
+                "response",
+            )
             if status_code:
-                result = "HTTP {}xx".format(int(status_code) // 100)
-                elasticapm.set_transaction_result(result, override=False)
+                try:
+                    result = "HTTP {}xx".format(int(status_code) // 100)
+                    elasticapm.set_transaction_result(result, override=False)
+                except ValueError:
+                    elasticapm.set_transaction_result("HTTP 2xx", override=False)
```

Comment:
could you add a comment here lining out what causes the `ValueError`?

Reply:
Done. Casting to an `int` throws a `ValueError` if it's unsuccessful.